### PR TITLE
Add release-1.6 to the testing matrix

### PR DESCRIPTION
**Pull request by smoshiur1237**
Same as title

### CODE REVIEW

**Review by Sunnatillo — COMMENTED** [review]
LGTM

File: docs/user-guide/src/version_support.md [hunk 1]
```diff
@@ -39,37 +41,36 @@ The compatability of IPAM and CAPM3 API versions with CAPI is discussed
...
+CAPM3 release it was tested with.
 
 All container images are published through the
 [Metal³ organization in Quay](https://quay.io/organization/metal3-io).
 Here are some examples:
 
-- quay.io/metal3-io/cluster-api-provider-metal3:v1.5.0
-- quay.io/metal3-io/baremetal-operator:v0.4.0
-- quay.io/metal3-io/ip-address-manager:v1.5.0
-- quay.io/metal3-io/ironic:capm3-v1.5.0
```
Comment (tuminoid) [inline docs/user-guide/src/version_support.md:71]: Why are you removing Ironic here?

File: docs/user-guide/src/version_support.md [hunk 2]
```diff
@@ -81,13 +82,15 @@ The table describes which branches/image-tags are tested in each periodic CI tes
 | ------------------------------------------------ | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
 | daily_main_integration_test_ubuntu/centos        | main         | main        | main           | latest         | latest      | latest     |
 | daily_main_e2e_integration_test_ubuntu/centos    | main         | main        | main           | latest         | latest      | latest     |
+| daily_release-1-6_integration_test_ubuntu/centos | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | latest     |
 | daily_release-1-5_integration_test_ubuntu/centos | release-1.5  | release-1.5 | release-0.4    | v0.4.0         | latest      | latest     |
 | daily_release-1-4_integration_test_ubuntu/centos | release-1.4  | release-1.4 | release-0.3    | v0.3.1         | latest      | latest     |
 | daily_release-1-3_integration_test_ubuntu/centos | release-1.3  | release-1.3 | v0.2.0         | v0.2.0         | latest      | latest     |
```
Comment (tuminoid) [inline docs/user-guide/src/version_support.md:88]: 1.3 jobs are being dropped from the JJB right now, might as well drop them in the same PR.

File: docs/user-guide/src/version_support.md (hunk 1 above)
Comment (smoshiur1237) [reply docs/user-guide/src/version_support.md:71]: There is no ironic tag release for v1.6.0, thats why I removed it

Comment (smoshiur1237) [reply docs/user-guide/src/version_support.md:71]: I am adding the irornic tag, thanks for the catch

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Review by kashifest — COMMENTED** [review]
/approve

**Pull request merged**